src/bilder/images/edxapp/custom_install.pkr.hcl

```diff
@@ -83,6 +83,7 @@ source "amazon-ebs" "edxapp" {
     OU              = var.business_unit
     app             = local.app_name
     deployment      = var.installation_target
+    framework       = "ansible"
```

Comment:
Probably worth adding this to the `base` template as well just for belt and suspenders.